packages/storybook/src/nimble/table/table.mdx

```diff
@@ -164,4 +164,18 @@ the current row have been traversed, `Tab`/`Shift-Tab` will move focus outside
 the table. When the table is focused again, navigation will resume from the
 previously focused element.
 
+### Selecting multiple rows
+
+When the table has a `selectionMode` of `multiple`, multiple rows can be
+selected using the selection checkboxes. Additionally, clicking on rows has the
+following behavior:
+
+-   Clicking on a row selects just that row
+-   `CTRL` clicking on a row toggles the selection of that row while maintaining
+    the selection of other rows
+-   Selecting a row followed by `SHIFT` clicking on another row selects the
+    range of rows between the two rows
+-   Selecting a row followed by `SHIFT-CTRL` clicking on a different row adds
+    the range of rows to the existing selection
```

Comment:
On macOS it's the Command key (`⌘`) rather than `CTRL`. Ctrl-click is treated as a right click.